src/testdir/test_breakindent.vim

```diff
@@ -1166,6 +1166,7 @@ func Test_breakindent_change_display_uhex()
 endfunc
 
 func Test_breakindent_list_split()
+  CheckScreendump
```

Comment:
?

Reply:
This looks like an oversight. I think it can be removed. Thanks for pointing it out.